### DISCUSSION TITLE
What does the lines actually mean?

### DISCUSSION THREAD

**Discussion by applejag** [remarkjs/remark#605] [Q&A] [answered]
Hi! I ran remark-cli and got the following output:

```
.github/ISSUE_TEMPLATE/feature_request.md
  2:1-7:4  warning  Headings should use atx         heading-style           remark-lint
```

I get the warning, but the `2:1-7:4`... What does that mean?

Is it on line 2, 1 to 7, 4? Is it on column 1-7?

Thanks in advance!

**Comment by wooorm** [comment, marked as answer]
Hi kalle! That depends a lot on your setup, the file you’re checking, and the versions of dependencies.

But, probably, because you have a setext heading, and the way you’ve configured remark-lint means that ATX headings are preferred.

See https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-heading-style for more info

**Reply by applejag** [reply]
Yea the issue was that I used so called frontmatters, and fixed it by also using the https://github.com/remarkjs/remark-frontmatter package.

But I'm still no wiser to what the numbers `2:1-7:4`  stood for.

**Reply by wooorm** [reply]
Line 2, column 1; line 7, column 4

**Reply by applejag** [reply]
Awesome! Thanks
